src/libraries/System.Private.CoreLib/src/System/Byte.cs

```diff
@@ -15,11 +15,11 @@ namespace System
     [TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public readonly struct Byte : IComparable, IConvertible, ISpanFormattable, IComparable<byte>, IEquatable<byte>
 #if FEATURE_GENERIC_MATH
-#pragma warning disable SA1001
+#pragma warning disable SA1001, CA2252
```

Comment:
```suggestion
#pragma warning disable SA1001, CA2252 // SA1001: Comma positioning; CA2252: Preview Features
```
With the same suggestion applied everywhere you have `#pragma warning disable`.